.github/workflows/go.yml

```
@@ -61,140 +61,140 @@ jobs:
+              uses: dawidd6/action-download-artifact@v9
+              with:
+                branch: master
+                workflow_conclusion: success
+                name: master.breakdown
+                if_no_artifact_found: warn
+
+            - name: download artifact (master-coverage.out)
+              id: download-master-coverage
+              uses: dawidd6/action-download-artifact@v9
+              with:
+                branch: master
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)